client/jetpack-connect/authorize-form.jsx

```diff
@@ -113,7 +113,7 @@ class JetpackConnectAuthorizeForm extends Component {
 					actionURL="/jetpack/connect"
 				/>
 				<LoggedOutFormLinks>
-					<JetpackConnectHappychatButton>
+					<JetpackConnectHappychatButton eventName="calypso_jpc_404_chat_initiated">
```

Comment:
Do you think we could figure a better name than `404` for this one? I don't think it's exactly a 404 case.